hieradata_aws/class/integration/ckan.yaml

```diff
@@ -3,5 +3,13 @@
 govuk_solr::disable: true
 govuk_solr6::present: true
 
+govuk::apps::ckan::blanket_redirect_url: https://find-data-beta-integration.cloudapps.digital/ckan_maintenance
```

Comment:
are you wanting a:

'govuk::apps::ckan::enabled: true'

somewhere here?

Reply:
It's currently working, not sure if we need to explicitly declare that

Reply:
ok cool!